### PR TITLE
Creating Safia profil

### PR DESCRIPTION
**Pull request by Safia11**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
3 words might be misspelled, please check them: CHMITI, login, Safia

**Comment by ElevenWilson** [comment]
3 words might be misspelled, please check them: CHMITI, login, Safia

File: _authors/Safia11.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: author
+login: Safia11
+title: Safia CHMITI
+twitter:
```
Comment (HugoDurand) [inline _authors/Safia11.md]: permalink

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Pull request merged**